db_replace.php

```diff
@@ -43,12 +48,23 @@
 echo $OUTPUT->header();
 if ($form->is_cancelled()) {
     redirect($redirect);
-} else if ($csvcontent = $form->get_file_content('csvfile')) {
-    helper::handle_replace_csv($csvcontent);
-} else {
+} else if (!isset($CFG->allowuireplace) || !$CFG->allowuireplace) {
```

Comment:
CFG->allowuireplace needs to be namespaced eg in config.php:

$CFG->forced_plugin_settings['tool_advancedreplace']['allowuireplace'] = 'duck';

and in code its normal get_config()

Reply:
Update with name and to use to get_config()